core/util/hyperparameter_baseline.py

```diff
@@ -0,0 +1,13 @@
+from core.util.loss_functions import zap_loss
+
+
+def get_hyperparamter_baseline() -> dict:
+    """Get dictionary of our hyperparameter baseline."""
+    return {
+        "hidden_size": 16,
+        "epochs": 10,
+        "horizon": 24,
+        "lookback": 36,
+        "loss_function": zap_loss,
+        "dropout_rate": 0.2,
+    }
```

Comment:
Could change this to hyperparameter_configuration, as baseline does not seem to convey the meaning. Also small typo in "hyperparamter" -> "hyperparameter"